php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2107,11 +2107,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      */
     function _nullScores(array $scoreCols): void
     {
-        $record    = $this->loadInstanceData($this);
-        $scoreCols = array_combine($scoreCols, $scoreCols);
-        // order is important here because it keeps values of first array
-        // unsetting all columns which are not scores
-        $data = array_intersect_key($record, $scoreCols);
+        $record = $this->loadInstanceData($this);
```

Comment:
The loadInstanceData seems to be the cause of this problem, and thinking through it, I'm not sure I understand why this needs to be here at all.

Can't we just null the scores in `$scoreCols` without adding the data from loadInstanceData?

Reply:
You're right, I'll make these changes. 
Thanks for your feedback !